snappy/snap_local.go

```diff
@@ -57,87 +59,115 @@ func NewInstalledSnap(yamlPath string) (*Snap, error) {
 	return snap, nil
 }
 
+func completeInfo(info *snap.Info, manifest *diskManifest) {
+	// XXX: we actually should not trust snap.yaml name!
+
+	info.Developer = SideloadedDeveloper
```

Comment:
we can try that but yes would prefer to try a bit later than the shuffling everything, we don't store in names anymore I think, so shouldn't be backward incompatible but may need integration tests work